api/envoy/config/route/v3/route_components.proto

```diff
@@ -839,6 +839,12 @@ message RouteAction {
     // :ref:`traffic splitting <config_http_conn_man_route_table_traffic_splitting_split>`
     // for additional documentation.
     WeightedCluster weighted_clusters = 3;
+
+    // [#not-implemented-hide:]
+    // Name of the cluster specifier plugin to use to determine the cluster for
+    // requests on this route. The plugin name must be defined in the associated
+    // :ref:`envoy_v3_api_msg_config.route.v3.RouteConfiguration.cluster_specifier_plugins`.
```

Comment:
nit: pedantically, can you mention that this string refers to the name inside the `TypedExtensionConfig`? It's not completely clear to the reader what this string refers to.

Reply:
FWIW I'm not 100% sure about the refs -- it _looks_ like `envoy_v3_api_msg_config` is used to link to a message and `envoy_v3_api_field_config` is used to link to a field within a message.  Is that right?

Also I'm having some trouble running `fix_format`, so this should not be merged until I've worked that out.  (I assume the presubmits will fail anyway.)